src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -387,6 +387,126 @@ instruct vadd_fp_masked(vReg dst_src1, vReg src2, vRegMask_V0 v0) %{
   ins_pipe(pipe_slow);
 %}
 
+// vector-immediate add (unpredicated)
+
+instruct vadd_immI(vReg dst_src, immI5 con) %{
+  match(Set dst_src (AddVB dst_src (Replicate con)));
+  match(Set dst_src (AddVS dst_src (Replicate con)));
+  match(Set dst_src (AddVI dst_src (Replicate con)));
```

Comment:
Is it necessary to require that `src` and `dst` be the same register for un-predicated versions?